collect_app/src/main/java/org/odk/collect/android/utilities/FormPageLifecyleOwner.kt

```diff
@@ -0,0 +1,23 @@
+package org.odk.collect.android.utilities
+
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LifecycleRegistry
+
+class FormPageLifecyleOwner : LifecycleOwner {
```

Comment:
I'm not sure that it makes sense to rename this. It's still a utility object that could be used in many different contexts and `DestroyableLifecycleOwner` makes sense with the new (more correct) behaviour in my opinion.

Reply:
Right as you said it could be used in many different contexts so it doesn't make sense to use that `FormPage` prefix. I've reverted that change but... is `Destroyable` a good one here? It's a normal class like many other so why `Destroyable`?

Reply:
I guess originally I named it that way because I needed a `LifecycleOwner` I could manually destroy (with the `destroy` method here) from the outside. Agreed it's not the best, but I'm now having trouble thinking of a better name. Any ideas?

Reply:
Yeah it's not easy in this case but maybe: `CustomLifecycleOwner`, `PlainLifecycleOwner`, `LifecycleOwnerImpl`... what do you think?

Reply:
How about `ControllableLifecycleOwner`. I think the main feature of this implementation is that the lifecycle can be "controlled" by a client (through `start`/`stop`) as opposed the `Activity`/`Fragment` where the lifecycle transitions are not **directly** controllable.

Reply:
Sounds better than `DestroyableLifecycleOwner` so yeah.